### DISCUSSION TITLE
考虑给自定义主页增加一个日志控件吗？

### DISCUSSION THREAD

**Discussion by CreeperIsASpy** [Meloong-Git/PCL#4590] [讨论]
比如local:MyLog之类的，这样自定义主页就可以植入pcl的日志了
随便问问。。

**Comment by Ad-closeNN** [comment]
这...不如开个调试模式呢

**Reply by CreeperIsASpy** [reply]
对，但是。。。
调试模式只能显示在最下层，占满全宽
对于我这种紧凑的主页肥肠不友好

**Comment by MoYuan-CN** [comment]
如果你需要的话，请开一个 Issue，尚未找到相关帖子
~~但这东西大概率不会做罢~~

**Comment by LTCatt** [comment]
不会做的……
